api/api_test.go

```diff
@@ -44,7 +45,7 @@ func TestTickers(t *testing.T) {
 	db := internal.InitRedis(fmt.Sprintf("redis://%s", s.Addr()))
 	seedDb(t, db)
 
-	Bootstrap(engine, db, getChartsMock(), getAssetClientMock())
+	Bootstrap(engine, db, getChartsMock(), getAssetClientMock(), nil)
```

Comment:
why special case the test?

Reply:
What do you mean ?

Reply:
We just reduce the time to init cache, we do not need it if we test another endpoint